### PR TITLE
Update RNChartViewBase.swift

### PR DESCRIPTION
**Pull request by danceyoung**
support setting limitline's properties:
	lineDashPhase
	lineDashLengths
	valueTextColor
	valueFont
	drawLabelEnabled
	labelPosition

### CODE REVIEW

File: ios/ReactNativeCharts/RNChartViewBase.swift [hunk 1]
```diff
@@ -277,19 +277,56 @@ open class RNChartViewBase: UIView, ChartViewDelegate {
...
+                    }
+                    
                     if limitLineConfig["lineWidth"].number != nil {
                         limitLine.lineWidth = CGFloat(limitLineConfig["lineWidth"].numberValue)
                     }
-                    
+                  
+                    if limitLineConfig["labelPosition"].string != nil {
+                      switch limitLineConfig["labelPosition"].stringValue {
+                      case "RIGHT_BOTTOM":
```
Comment (wuxudong) [inline ios/ReactNativeCharts/RNChartViewBase.swift]: move enum parsing to BridgeUtils will be better.

File: ios/ReactNativeCharts/RNChartViewBase.swift [hunk 2]
```diff
@@ -277,19 +277,56 @@ open class RNChartViewBase: UIView, ChartViewDelegate {
                 if limitLineConfig["limit"].double != nil {
```
Comment (wuxudong) [inline ios/ReactNativeCharts/RNChartViewBase.swift]: Can you also add an android implementation for it? Keep android and iOS in consistent is important.

Comment (danceyoung) [reply ios/ReactNativeCharts/RNChartViewBase.swift]: ok

File: ios/ReactNativeCharts/RNChartViewBase.swift (hunk 1 above)
Comment (danceyoung) [reply ios/ReactNativeCharts/RNChartViewBase.swift]: I see

**Comment by danceyoung** [comment]
env:
    "react": "16.0.0",
    "react-native": "0.51.0",
    "react-native-charts-wrapper": "0.4.1",

**Pull request merged**